recipe/meta.yaml

```diff
@@ -29,28 +29,30 @@ requirements:
     - dask
     - entrypoints
     - fsspec
-    - fiona ==1.8.22
-    - gdal >=3.1
     - geopandas >=0.10
-    - netcdf4
-    - numpy >=1.20
+    - mercantile
     - numba
+    - numpy >=1.20
+    - netcdf4
     - openpyxl
     - packaging
     - pandas
     - pyflwdir >=0.5.4
     - pyproj
     - python >=3.9
+    - pyyaml
```

Comment:
I think we are missing pyyaml in our pyproject.toml, no? This should also be added there I think.